### PR TITLE
All Tests required for SAAS-819

### PR DESCRIPTION
**Pull request by peterSirotnak**
Pull request for story: [SAAS-819](https://jira.percona.com/browse/SAAS-819)

### CODE REVIEW

File: cypress/integration/freeUser/organizations/members.spec.js [hunk 1]
```diff
@@ -0,0 +1,104 @@
...
+        cy.contains('td', admin1User.email)
+            .parent()
+            .within(() => cy.findByTestId(organizationPage.locators.editMemberIcon).should('be.disabled'));
+        const emails = [technical1User.email, admin2User.email];
+        const oldValues = ['Technical User', 'Admin'];
+        const newValues = ['Admin', 'Technical'];
+
+        // Change user roles from Technical to Admin and Admin to Technical
+        // eslint-disable-next-line no-plusplus, no-magic-numbers
+        for (let i = 0; i < emails.length; i++) {
```
Comment (yurkovychv) [inline cypress/integration/freeUser/organizations/members.spec.js:41]: could you please use [forEach](https://developer.mozilla.org/ru/docs/Web/JavaScript/Reference/Global_Objects/Array/forEach) here?

File: cypress/integration/freeUser/organizations/members.spec.js [hunk 2]
```diff
@@ -0,0 +1,104 @@
...
+            cy.findByTestId(organizationPage.locators.ediMemberSubmitButton).click();
+            cy.contains('td', emails[i])
+                .parent()
+                .then((row) => {
+                    expect(row).to.deep.contain(newValues[i]);
+                });
+            cy.checkPopUpMessage(organizationPage.constants.messages.memberEditedSuccessfully);
+        }
+
+        cy.removeCurrentUserAccessToken();
```
Comment (yurkovychv) [inline cypress/integration/freeUser/organizations/members.spec.js:57]: I think this is not needed in the end of a test

File: cypress/integration/freeUser/organizations/members.spec.js [hunk 3]
```diff
@@ -0,0 +1,104 @@
...
+            // Edit button for Technical user in user should be disabled for all users.
+            cy.contains('td', user.Email)
+                .parent()
+                .within(() => {
+                    cy.findByTestId(organizationPage.locators.editMemberIcon).should('be.disabled');
+                    cy.findByTestId(organizationPage.locators.deleteMemberIcon).should('be.disabled');
+                });
+        });
+
+        cy.removeCurrentUserAccessToken();
```
Comment (yurkovychv) [inline cypress/integration/freeUser/organizations/members.spec.js:102]: same here

File: cypress/integration/freeUser/organizations/members.spec.js (hunk 1 above)
Comment (peterSirotnak) [reply cypress/integration/freeUser/organizations/members.spec.js:41]: I do need i value for newValues[i], oldValues[i] to be able to compare old and new value for user role.

File: cypress/integration/freeUser/organizations/members.spec.js (hunk 2 above)
Comment (peterSirotnak) [reply cypress/integration/freeUser/organizations/members.spec.js:56]: We left it in previous pull requests, but if you want to I can move it to AfterEach() method.

File: cypress/integration/freeUser/organizations/members.spec.js (hunk 3 above)
Comment (peterSirotnak) [reply cypress/integration/freeUser/organizations/members.spec.js:101]: Same as previous comment

File: cypress/integration/freeUser/organizations/members.spec.js (hunk 2 above)
Comment (yurkovychv) [reply cypress/integration/freeUser/organizations/members.spec.js:56]: @peterSirotnak cypress automatically clears all browser data between test. https://docs.cypress.io/api/commands/clearlocalstorage

**Review by yurkovychv — APPROVED** [review]

**Pull request merged**